t/sanity.t

```diff
@@ -815,7 +815,7 @@ init_worker_by_lua '
         shm = "healthcheck",
         upstream = "foo.com",
         type = "http",
-        http_req = "GET /status HTTP/1.0\\\\r\\\\nHost: localhost\\\\r\\\\n\\\\r\\\\n",
+        http_req = "GET /status HTTP/1.0\r\nHost: localhost\r\n\r\n",
```

Comment:
So which one is correct here?
http_req = "GET /status HTTP/1.0\r\nHost: localhost\r\n\r\n" (like line 1130)
or
http_req = "GET /status HTTP/1.0\\\\r\\\\nHost: localhost\\\\r\\\\n\\\\r\\\\n" (like line 1035)